### ISSUE TITLE
Retries on PT API access

### ISSUE THREAD

**Issue by nick-ratbert** [Legedric/ptmagic#507]
I have a Linux VPS which uses pm2 to restart processes over a reboot.

PT, PTM and Monitor all start at about the same time.

PT needs some time to get ready, so it's API is not immediately available. PTM appears to try only once to access the API and then just sits there doing nothing obvious.

Could it be made to retry every few seconds?

My server rebooted a few days ago. PTv2 and Monitor all ran fine but, unusually, I had no messages from PTM. When I went and looked, it was running, but the logs only said the API was unavailable. I restarted PTM and everything was working fine.

Thanks

**Comment by Legedric** [comment]
Duplicate of #457
